app/views/beaches/show.html.erb

```diff
@@ -33,8 +33,15 @@
 </p>
 
 <% @reviews.each do |review| %>
-  <%= review.rating %>
-  <%= review.description %>
+  <li><%= review.rating %></li>
+  <% unless review.description.nil? %>
+    <li><%= review.description %></li>
+  <% end %>
+  <li><%= link_to "Delete Review", "/beaches/#{@beach.id}/reviews/#{review.id}",
```

Comment:
`link_to` defaults to a GET request. You can pass another method to it override the default:

```ruby
link_to "Delete this thing", thing_path(thing), method: :delete
```